index.html

```diff
@@ -0,0 +1,865 @@
+                  <p class="h6 font-weight-bolder">
+                    Does joe biden secretly worship kittens?
+                  </p>
+                  <p class="small font-weight-bolder text-secondary">
+                    ASKS THE NEKO MASTER
+                  </p>
+                </div>
+              </div>
+              <div class="d-flex flex-row justify-content-start border-bottom my-2 py-2">
+                <img src="https://placekitten.com/200/283" class="rounded-circle mr-2" width="70" height="65"
+                  alt="kitty" />
+                <div class="d-flex flex-column justify-content-center">
+                  <p class="h6 font-weight-bolder">Did the cats build the pyramids?</p>
+                  <p class="small font-weight-bolder text-secondary">
+                    ASKS THE NEKO MASTER
+                  </p>
+                </div>
+              </div>
+              <div class="d-flex flex-row justify-content-start my-2 py-2">
+                <img src="https://placekitten.com/200/283" class="rounded-circle mr-2" width="70" height="65"
+                  alt="kitty" />
+                <div class="d-flex flex-column justify-content-center">
+                  <p class="h6 font-weight-bolder">
+                    Kittens threatens to overtake the planet!
+                  </p>
+                  <p class="small font-weight-bolder text-secondary">
+                    CONFIRMS THE NEKO MASTER
+                  </p>
+                </div>
+
+              </div>
+              <div class="row justify-content-center">
+                <form action="/action_page.php"
+                  class="d-none d-md-flex flex-column  bg-light px-3 py-4 border mx-3 col-5 form-lg">
+                  <div class="d-flex flex-row align-items-center mb-3">
+                    <img class="mr-2 img-icon" src="https://placekitten.com/96/139" alt="kitty" width="35"
+                      height="35" />
+                    <h6 class="font-weight-bolder mt-1">
+                      START YOUR DAY WITH OUR TOP 5 CUTE KITTENS
+                    </h6>
+                  </div>
+                  <div class="form-group">
+                    <input type="email" class="form-control" placeholder="Enter email" id="email2" />
+                  </div>
+                  <button type="submit" class="btn color-reverse">FREE SIGN UP</button>
+                </form>
+              </div>
+            </div>
+          </div>
+    </div>
+  </div>
+
+  <div class="container-fluid">
+    <div class="row no-glutters mx-1">
+       <div class="col-12 bg-white">
+         <div class="article bg-white pr-3 pl-3 pt-3 pb-1">
+        <div class="d-flex justify-content-center align-items-center mb-3 border-bottom pb-3">
+          <span class="font-weight-bold">
+            <img class="mr-2" src="https://placekitten.com/g/200/300" alt="Magazine" /> IN
+            THE MAGAZINE
+          </span>
```

Comment:
remember: your building a clone stick with the original design

![image](https://user-images.githubusercontent.com/51195150/99135148-39c63780-2621-11eb-9599-568a62cb8740.png)